src/test/system_tests/ceph_s3_tests/test_ceph_s3_deploy.sh

```diff
@@ -24,6 +24,14 @@ if [ ! -d $DIRECTORY ]; then
     cd ${DIRECTORY}
     git checkout ${CEPH_TESTS_VERSION}
     echo "Finished Downloading Ceph S3 Tests"
+
+    #s3 tests for select uses the hard-coded bucket name "test".
+    #The automatic teardown that deletes buckets created by tests expects a name
+    #with a certain pattern like the bucket names get_new_bucket_name() generates.
+    #The following manual fix will be obsolete if and when https://github.com/ceph/s3-tests/pull/488 is merged.
```

Comment:
Great explanation, thank you.
Please add a space after the "#" sign in every line to match the original style of comment in the file.